HeliosEmbedded/Makefile

```diff
@@ -270,5 +283,20 @@ extract_hex: helios_firmware.hex
 clean:
 	rm -f $(OBJS) $(TARGET).elf $(TARGET).hex $(DFILES) $(TARGET).bin $(TARGET).eep $(TARGET).lst $(TARGET).map
 
+compute_version:
+	$(eval LATEST_TAG ?= $(shell git fetch --depth=1 origin +refs/tags/*:refs/tags/* &> /dev/null && git tag --list "*$(BRANCH_SUFFIX)" | sort -V | tail -n1))

```

Comment:
the logic for $(BRANCH_SUFFIX) is unnecessary here, you can just do 
git tag --list | sort...

that is for filtering the various branches of vortex engine (duo, gloves, handle, etc)